<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>From'Sondage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="startWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="surveyButton">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Remplir un sondage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statButton">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Acc√©der aux statistiques</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
